### PR TITLE
basic test

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1 +1,23 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
```
Comment (Kasatello) [inline app/test_main.py:3]: there should be an empty blank lane between built-in modules and local

**Review by Kasatello — APPROVED** [review]

File: app/test_main.py [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+
+@pytest.mark.parametrize(
+    "url, has_internet, is_valid, expected",
+    [
+        ("https://www.google.com", True, True, "Accessible"),
+        ("https://www.google.com", False, True, "Not accessible"),
+        ("https://somewebsitename.com", True, False, "Not accessible"),
+        ("https://somewebsitename.com", False, False, "Not accessible")
```
Comment (YuriiKindrat) [inline app/test_main.py:12]: it seems to me there is no need to use different urls for this tests, so u can write an url only once directly in ur func

File: app/test_main.py (hunk 1 above)
Comment (YuriiKindrat) [reply app/test_main.py:3]: agree, according to pep8 
https://peps.python.org/pep-0008/#module-level-dunder-names:~:text=Imports%20should%20be,group%20of%20imports.

File: app/test_main.py [hunk 3]
```diff
@@ -1 +1,23 @@
...
+    ]
+)
+def test_can_access_google_page(
+    url: str, has_internet: bool, is_valid: bool, expected: str
+) -> None:
+    with mock.patch(
+            "app.main.has_internet_connection",
+            return_value=has_internet
+    ):
+        with mock.patch("app.main.valid_google_url", return_value=is_valid):
```
Comment (YuriiKindrat) [inline app/test_main.py:22]: no need to use `with` twice here, u can use it only once as for `open` function

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Review by YuriiKindrat — APPROVED** [review]

**Pull request closed without merging**